lib/core-server/src/utils/stories-json.ts

```diff
@@ -80,4 +81,31 @@ export async function useStoriesJson(
       res.send(err.message);
     }
   });
+
+  router.use('/stories.json', async (req: Request, res: Response) => {
```

Comment:
Are we going to write this on build too? That seems more crucial (composition with v6 storybooks).

Reply:
We already do